industrial_ci/src/ci_main.sh

```diff
@@ -49,6 +49,10 @@ fi
 if [ "${AFTER_SCRIPT// }" != "" ]; then
   ici_time_start after_script
 
+  if [ -z "$DOCKER_COMMIT" ]; then
```

Comment:
Will only get executed if `AFTER_SCRIPT` is defined.

Reply:
and `DOCKER_COMMIT` gets unset before the the script executions anyway.